deploy/examples/create-external-cluster-resources.py

```diff
@@ -1829,6 +1921,35 @@ def gen_json_out(self):
                     }
                 )
 
+        # if 'TOPOLOGY_POOLS', 'TOPOLOGY_FAILURE_DOMAIN_LABEL', 'TOPOLOGY_FAILURE_DOMAIN_VALUES'  exists,
+        # then only add 'topology' StorageClass
+        if (
+            self.out_map["TOPOLOGY_POOLS"]
+            and self.out_map["TOPOLOGY_FAILURE_DOMAIN_LABEL"]
+            and self.out_map["TOPOLOGY_FAILURE_DOMAIN_VALUES"]
+        ):
+            json_out.append(
+                {
+                    "name": "ceph-rbd-topology-storageclass",
```

Comment:
Looking at the naming convention of the other storageclass like ceph-rbd, ceph-rbd-rados-namespace, cephfs, ceph-rgw, shouldn't this storageclass be called just "ceph-rbd-topology" instead of "ceph-rbd-topology-storageclass"?